samples/highcharts/blog/simple-Logarithmic-graph/demo.js

```diff
@@ -0,0 +1,56 @@
+Highcharts.chart("container", {
```

Comment:
This file doesn't lint because it has indenting of 2 spaces. It should be blocked for committing. Did you run `npm i` before commiting?